docs/content/guides/developer/getting-started/graphql-rpc.mdx

```diff
@@ -12,41 +12,117 @@ keywords: [ rpc, sui rpc, graphql, how to use graphql, schema, graphql schema, q
 
 :::
 
-The quickest way to access the GraphQL service for Sui RPC is through the online IDE that provides a complete toolbox for fetching data and executing transactions on the network. The online IDE provides features such as auto-completion (use Ctrl+Space or just start typing), built-in documentation (Book icon, top-left), multi-tabs, and more.
+The quickest way to access the GraphQL service for Sui RPC is through the online IDE that provides a complete toolbox for fetching data and executing transactions on the network. The online IDE provides features such as auto-completion (use <kbd>Ctrl</kbd>+<kbd>Space</kbd> or just start typing), built-in documentation (Book icon, top-left), multi-tabs, and more.
 
-The online IDE is available for [Mainnet](https://sui-mainnet.mystenlabs.com/graphql) and [Testnet](https://sui-testnet.mystenlabs.com/graphql). This guide contains various queries that you can try directly in the IDE.
+The online IDE is available for Mainnet and Testnet. You can try the various queries on this page directly in the IDE.
+
+- Mainnet: https://graphql.mainnet.sui.io/graphql
+- Testnet: https://graphql.testnet.sui.io/graphql
```

Comment:
Can we add a info or warn admonition here that developers should stop using the GraphQL alpha endpoints from earlier - https://sui-mainnet.mystenlabs.com/graphql and https://sui-testnet.mystenlabs.com/graphql, and that they should migrate their apps to use the new beta schema as soon as possible?